client/src/elm/Translate.elm

```diff
@@ -15110,6 +15184,26 @@ translateActivePage page =
                     , kinyarwanda = Nothing
                     }
 
+                NCDParticipantPage _ _ ->
+                    { english = "NCD Encounter"
```

Comment:
Is this label correct?

Reply:
Yes, it;'s correct. We show this 'Encounter' on Participant page - same af for other types of participants.